### PR TITLE
Repatriate the ASP.NET Core SDK under the monorepo

### PR DESCRIPTION
**Pull request by Tyrrrz**
As far as I know, the ASP.NET Core SDK was split out from the previous monorepo, which included Passwordless.Server. While it makes sense for the Server repo to be separate from the SDK projects, keeping the SDKs together provides us with a lot of benefits:

- We can use the same exact CI workflow without any additions or changes
- Test and deploy flows will work exactly the same for both
- Dependent SDKs (`.AspNetCore` in this case) will always be tested and released against the latest version of the core SDK
- Better discoverability and issue management: users won't have to figure out where to report their bugs
- We can share tooling (`.editorconfig` etc) instead of copy-pasting

### CODE REVIEW

**Comment by Tyrrrz** [comment]
Note that the current GitHub token provided by the CI does not have permissions to push `Passwordless.AspNetCore` to the registry. The same problem will likely occur with the NuGet token too, I'm not sure what scopes are set there.

File: src/Passwordless/Passwordless.csproj [hunk 1]
```diff
@@ -45,11 +50,6 @@
     <PackageReference Include="System.Net.Http.Json" Version="7.0.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="../../favicon.png" Pack="true" PackagePath="/" Visible="false" />
-    <None Include="../../README.md" Pack="true" PackagePath="/" Visible="false" />
-  </ItemGroup>
```
Comment (abergs) [inline src/Passwordless/Passwordless.csproj:51]: Why is this removed while it seems to be kept in the AspNetCore csproj?

Comment (Tyrrrz) [reply src/Passwordless/Passwordless.csproj:51]: I actually just moved it to the top of the file for consistency. It's still there.

**Comment by justindbaur** [comment]
> add our examples into the same repo

Yes please, just have an `examples` directory.

**Comment by Tyrrrz** [comment]
Added the example @abergs @justindbaur

It references the SDK by PackageReference, not ProjectReference, like Anders suggested, so that users can still easily copy-paste the sources.
